README.md

```diff
@@ -72,11 +73,38 @@ Tha last step is to install the service and remove the downloaded files:
 
 ```bash
 /data/dbus-opendtu/install.sh
+/data/dbus-opendtu/restart.sh
 rm main.zip
 ```
 
 Check configuration after that - because the service is already installed and running. In case of wrong connection data (host, username, pwd) you will spam the log-file! Also, check to **set a** proper (minimal) **log level**
 
+### Update the code
+Just grap a copy of the main branch and copy the content to `/data/` e.g. `/data/dbus-opendtu`. The process will preserve your existing config ini. 
+After that call the `install.sh script.
+
+```bash
+wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/heads/main.zip
+unzip main.zip "dbus-opendtu-main/*" -d /data
+rm /data/dbus-opendtu/current.log*
+mv /data/dbus-opendtu-main/config.ini /data/dbus-opendtu-main/config.template
+cp -R /data/dbus-opendtu-main/* /data/dbus-opendtu
+rm -rf /data/dbus-opendtu-main/
+cp /data/dbus-opendtu/config.ini /data/dbus-opendtu/config.backup
+chmod a+x /data/dbus-opendtu/install.sh
+```
+
+Tha last step is to install the service and remove the downloaded files:
+
+```bash
+/data/dbus-opendtu/uninstall.sh
+/data/dbus-opendtu/install.sh
+/data/dbus-opendtu/restart.sh
+rm main.zip
+```
```

Comment:
(Where) should we add a "final" `rm -i /data/dbus-opendtu/current.log*` ?

Reply:
If you add it in the install and uninstall script you won't need it to add it elsewhere, if the install script is executed on an update.

Reply:
I am talking about our old log file location. This has to be removed once and never again

Reply:
Yes, but you don't know when an user updates your script. One tomorrow, one in a month, one in a year. Therefore also the check if the logfile exists.

Reply:
This is now in the install and uninstall scripts